src/styles/blocks/header/header.scss

```diff
@@ -0,0 +1,15 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.home-link {

```

Comment:
Is it separate BEM block? Need to check correct class name, I suppose it should be the element of other BEM block